pytorch_lightning/callbacks/finetuning.py

```diff
@@ -366,12 +346,12 @@ def __init__(
         self.initial_denom_lr: float = initial_denom_lr
         self.train_bn: bool = train_bn
         self.verbose: bool = verbose
-        self.round: int = round
+        self.rounding: int = rounding
         self.previous_backbone_lr: Optional[float] = None
 
     def on_save_checkpoint(
         self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", checkpoint: Dict[str, Any]
-    ) -> Dict[int, Any]:
+    ) -> Dict[str, Any]:
```

Comment:
```suggestion
    ) -> dict:
```
doesn't necessarily be string key.